electrum/fee_policy.py

```diff
@@ -0,0 +1,369 @@
+from typing import Optional, Sequence, Tuple, Union, TYPE_CHECKING
+from decimal import Decimal
+from numbers import Real
+from enum import IntEnum
+
+from .i18n import _
+from .util import NoDynamicFeeEstimates, quantize_feerate, format_fee_satoshis
+from . import util
+from . import constants
+
+if TYPE_CHECKING:
+    from .network import Network
+
+FEE_ETA_TARGETS = [25, 10, 5, 2, 1]
+FEE_DEPTH_TARGETS = [10_000_000, 5_000_000, 2_000_000, 1_000_000,
+                     800_000, 600_000, 400_000, 250_000, 100_000]
+FEERATE_STATIC_VALUES = [1000, 2000, 5000, 10000, 20000, 30000,
+                         50000, 70000, 100000, 150000, 200000, 300000]
+
+# satoshi per kbyte
+FEERATE_MAX_DYNAMIC = 1500000
+FEERATE_WARNING_HIGH_FEE = 600000
+FEERATE_FALLBACK_STATIC_FEE = 150000
+FEERATE_DEFAULT_RELAY = 1000
+FEERATE_MAX_RELAY = 50000
+
+# warn user if fee/amount for on-chain tx is higher than this
+FEE_RATIO_HIGH_WARNING = 0.05
+
+
+def closest_index(value, array) -> int:
+    dist = list(map(lambda x: abs(x - value), array))
+    return min(range(len(dist)), key=dist.__getitem__)
+
+
+class FeeMethod(IntEnum):
+    FIXED = 0
+    STATIC = 1
```

Comment:
please at least add some comments, the names are not clear:
```suggestion
    FIXED = 0  # fixed absolute fee
    STATIC = 1  # static fee rate
```
perhaps we should even rename them.
e.g. `STATIC` -> `STATICRATE`
or `FIXED` -> `ABS`